### ISSUE TITLE
Script Crash

### ISSUE THREAD

**Issue by cigarplug** [bluezio/ipwebcam-gst#61]
Edited:

I can see the video when I run the troubleshooting command:

> gst-launch-1.0 souphttpsrc location="http://192.168.1.115:9999/videofeed" \
>                do-timestamp=true is-live=true \
>     ! multipartdemux ! jpegdec ! videoconvert ! ximagesink

But when I use the 'dummy video device' in Zoom meeting app or Firefox, the video feed looks like this:
![Screenshot from 2020-03-23 13-54-47](https://user-images.githubusercontent.com/24826363/77276096-f293db00-6d0d-11ea-90e6-9bb2a91fc6e2.png)

**Comment by luco5826** [comment]
Same problem here, running Debian 10, kernel 4.19.0-9
Works flawlessly with the command above but on teams (based on skype) the Dummy video device returns the same image.
`v4l2l-ctl --list-device` correctly shows the loop back device:
```
Dummy video device (0x0000) (platform:v4l2loopback-000):
	/dev/video2
```
I love the idea of this app, so I'd be glad to help if possible

**Comment by bluezio** [comment]
What is the error output you get from the script?

**Comment by luco5826** [comment]
On my machine, nothing from the shell but a dialog appear like this 

![immagine](https://user-images.githubusercontent.com/38291180/77416301-0437ba00-6dc4-11ea-966a-8446f266de96.png)

And up until yesterday it was showing but not working. Right now I re-did it and not it works I did not change anything, I just shut down my PC and turned it back on today. 
Maybe could be the re-boot that fixed the issue? I saw another issue noticing the same "fix"
